manifests/0000_50_cluster_monitoring_operator_02-role.yaml

```diff
@@ -163,6 +170,15 @@ rules:
   - storage.k8s.io
   resources:
   - storageclasses
+  - volumeattachments
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  - validatingwebhookconfigurations
```

Comment:
out of curiosity: do we also need those permissions inherited?

Reply:
These are new metrics based on those resources added to kube-state-metrics. We are in the process of switching to this version of ksm yes.